Thu Dec 29 19:03:10 2022 - INFO  - start to run testcase:oe_test_auter.
Thu Dec 29 19:10:40 2022 - ERROR - The case exit by code 4.
Thu Dec 29 19:10:42 2022 - INFO  - End to run testcase:oe_test_auter.
Thu Dec 29 19:10:43 2022 - INFO  - A total of 1 use cases were executed, with 0 successes and 1 failures.
